core-web/apps/dotcms-ui/src/stories/primeng/form/TreeSelect.stories.ts

```diff
@@ -0,0 +1,272 @@
+        },
+        template: `
+        <div class="card flex justify-content-center w-25rem h-25rem">
+            <div class="md:w-20rem w-full">
+                <span class="p-float-label w-full">
+                    <p-treeSelect
+                        containerStyleClass="w-full"
+                        [(ngModel)]="selectedNodes"
+                        [options]="files"
+                        placeholder="Select Item"
+                        [showClear]="true"
+                    >
+                        <ng-template pTemplate="triggericon">
+                            <i class="pi pi-chevron-down"></i>
+                        </ng-template>
+                    </p-treeSelect> 
+                    <label for="treeselect">Float Label</label>
+                </span>
+            </div>
+        </div>
+        `
+    })
+};
+
+export const Disable: Story = {
+    render: () => ({
+        moduleMetadata: {
+            imports: [TreeSelectModule, FormsModule, BrowserAnimationsModule, BrowserModule]
+        },
+        props: {
+            files,
+            selectedNodes: ''
+        },
+        template: `
+        <div class="card flex justify-content-center w-25rem h-25rem">
+            <p-treeSelect
+                [disabled]="true"
+                [(ngModel)]="selectedNodes"
+                class="md:w-20rem w-full" containerStyleClass="w-full"
+                [options]="files"
+                placeholder="Select Item"
+                [showClear]="true"
+            >
+                <ng-template pTemplate="triggericon">
+                    <i class="pi pi-chevron-down"></i>
+                </ng-template>
+            </p-treeSelect> 
+        </div>
+        `
+    })
+};
+
+export const Multiple: Story = {
+    render: () => ({
+        moduleMetadata: {
+            imports: [TreeSelectModule, FormsModule, BrowserAnimationsModule, BrowserModule]
+        },
+        props: {
+            files,
+            selectedNodes: ''
+        },
+        template: `  
+        <div class="card flex justify-content-center w-25rem h-25rem">
+            <p-treeSelect
+                class="w-full md:w-20rem"
+                containerStyleClass="w-full"
+                [(ngModel)]="selectedNodes"
+                [options]="files"
+                [metaKeySelection]="false"
+                selectionMode="multiple"
+                placeholder="Select Item"
+                [showClear]="true"
+            >
+                <ng-template pTemplate="triggericon">
+                    <i class="pi pi-chevron-down"></i>
+                </ng-template>
+            </p-treeSelect> 
+        </div>
+        `
+    })
+};
+
+
+export const Checkbox: Story = {
+    render: () => ({
+        moduleMetadata: {
+            imports: [TreeSelectModule, FormsModule, BrowserAnimationsModule, BrowserModule]
+        },
+        props: {
+            files,
+            selectedNodes: ''
+        },
```

Comment:
@zJaaal Ok great tip I will create a base config for all stories, to be honest we should use `moduleMetadata ` function of StoryBook to avoid repeating this block but when I use this function our StoryBook crashes in another PR I will tackle this problem. 

![Screenshot 2024-05-28 at 4 31 58 PM](https://github.com/dotCMS/core/assets/7611944/316148b8-5bfe-4fde-b65e-2d6d949fc4bb)